swsnu/swpp18-team12
# Feature/club detail front

#### What does this PR do?
Implements club pages for frontend
#### Description of Task to be completed?
- new club page
  > Form validation is needed, image upload needed
- put detail page
  > Form validation is needed, image upload needed
- post rating page
  > rating with form. need to make star shaped rating
- detail tab
  > need to alter design
- rating tab
  > need to alter design
- search page
  > casual search page. Pagination is not implemented
#### How should this be manually tested?
1. Open backend server at proper branch
2. Open frontend server
3. Login
4. Create a new club at `/club/new`
5. Search the created club at `/search`
6. Click 'see more' button to route to detail page. (`/club/:id`)
7. Post a rating by clicking the 'rate this club' button
8. Change club detail by clicking 'set details' button
